transactions/add-account/2024/aug-27/create_account.cdc

```diff
@@ -0,0 +1,38 @@
+import FlowToken from 0x1654653399040a61
+import FungibleToken from 0xf233dcee88fe0abe
+
+transaction(publicKeys: [String]) {
+
+	prepare(signer: AuthAccount) {
+		let acct = AuthAccount(payer: signer)
+
+		for i, publicKey in publicKeys {
+			let key = PublicKey(
+				publicKey: publicKey.decodeHex(),
+				signatureAlgorithm: SignatureAlgorithm.ECDSA_P256,
+			)
+
+			acct.keys.add(
+				publicKey: key,
+				hashAlgorithm: HashAlgorithm.SHA3_256,
```

Comment:
Not sure if these are these are KMS keys. if so, the hash algorithm should be SHA2_256
```suggestion
				hashAlgorithm: HashAlgorithm.SHA2_256,
```